Makefile

```diff
@@ -30,6 +30,10 @@ docs:
 test:
 	tox
 
+fmt:
+	reorder-python-imports --py3-plus --separate-from-import --separate-relative $(shell find baseplate/ tests/ -name '*.py')
```

Comment:
should the `--py3-plus --separate-from-import --separate-relative` parts be split out into a variable so we're not duplicating them between targets?

also, i think we need the flag for exiting with 0 even on changes on this one?

Reply:
> should the --py3-plus --separate-from-import --separate-relative parts be split out into a variable so we're not duplicating them between targets?

Makes sense.

> also, i think we need the flag for exiting with 0 even on changes on this one?

I don't think we need it, would we want it though?

Reply:
We'd want it (or starting the line with `-` to ignore the return code) so that if `reorder-python-imports` and `black` both have jobs to do when we run `make fmt` we don't just stop immediately after RPI did its thing because it returned an "error" code.

Reply:
Oops, I think my brain was a little fried, I thought that flag was already in there (I added it in another repo I was working on) and read your comment as the opposite of what you actually said.